src/util/futurepool.rs

```diff
@@ -22,11 +22,28 @@ use std::thread;
 use std::time::Duration;
 use futures::Future;
 use futures_cpupool::{self as cpupool, CpuFuture, CpuPool};
+use prometheus::{Counter, CounterVec, Gauge, GaugeVec};
 
 use util;
 use util::time::Instant;
 use util::collections::HashMap;
 
+lazy_static! {
+    pub static ref FUTUREPOOL_RUNNING_TASK_VEC: GaugeVec =
```

Comment:
like Worker, maybe using Pending and Handled is better?

Reply:
ok, previously I thought the "pending" in worker should be "pending" only.. so that I use different names.. and I checked again the code found out that "pending" in worker has the same semantic (actually pending + running).